### ISSUE TITLE
Trying to compile source

### ISSUE THREAD

**Issue by IntyLab** [stella-emu/stella#926]
I fetched the source using the "git clone" command, ran the Configure script, and tried to compile the source using "make -f Makefile".  Compilation halted on an error because the file "sys/termios.h" is not found.  This happened in src/os/unix/SerialPortUNIX.cxx at line 22.

I ran the command "find ./src -name termios.h" and got no results.  Is the file just missing from the source package?  Thanks for your help on this.

**Comment by skitt** [comment]
What platform are you building on? `sys/termios.h` is supposed to be provided by the C library.

**Comment by sa666666** [comment]
Yes, sounds like you don't have the C header files installed.  If it's for Ubuntu, perhaps install `build-essentials`?
